ladybug/cli/translate.py

```diff
@@ -101,9 +102,14 @@ def wea_to_constant(wea_file, value, output_file):
 
```

Comment:
Also, you should update the docstrings of the command to make it clear that the input can either be a Wea or an EPW.